src/program.rs

```diff
@@ -27,13 +27,14 @@ use crate::eval::cache::Cache as EvalCache;
 use crate::eval::VirtualMachine;
 use crate::identifier::Ident;
 use crate::term::{record::Field, RichTerm};
+use clap::ValueEnum;
 use codespan::FileId;
 use codespan_reporting::term::termcolor::{Ansi, ColorChoice, StandardStream};
 use std::ffi::OsString;
 use std::io::{self, Cursor, Read};
 use std::result::Result;
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, ValueEnum)]
```

Comment:
clap exposes a [`ColorChoice`](https://docs.rs/clap/latest/clap/enum.ColorChoice.html) type which does the same job as `ColorOpt`. we could possibly just use `ColorChoice` everywhere, though we need to provide mappings into `rustyline::config::ColorMode` and `termcolor::ColorChoice`, which we currently do with `From` impls, so it might also make sense to newtype it.

If nothing else, I'm pretty sure we can delete the `FromStr` impl below as it was only there for `structopt` compatibility.

Reply:
Done in 9af6fa7a